packages/sparkify/jest.config.ts

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable */
+export default {
+  displayName: 'sparkify',
+  preset: '../../jest.preset.js',
+  transform: {
+    '^.+\\.[tj]s$': ['ts-jest', { tsconfig: '<rootDir>/tsconfig.spec.json' }],
+  },
+  moduleFileExtensions: ['ts', 'js', 'html'],
+  coverageDirectory: '../../coverage/packages/sparkify',
```

Comment:
This configuration throws an error. I solved it by [moving it to the global configuration](https://github.com/seb-neo/sebspark/commit/857642d069694f172ad13e2cafdc00b6265f109e), so you can remove this line.

```suggestion
```